src/node.cc

```diff
@@ -3334,6 +3338,11 @@ void SetupProcessObject(Environment* env,
     READONLY_PROPERTY(process, "_debugWaitConnect", True(env->isolate()));
   }
 
+  // --expose_internals,--expose-internals
+  if (expose_internals) {
+    READONLY_PROPERTY(process, "_exposeInternals", True(env->isolate()));
+  }
```

Comment:
PTAL, there is *no* new property created off of `process` (its deleted later). I'll add an assertion of that fact to the existing test.